readthedocs/projects/tasks.py

```diff
@@ -621,6 +621,11 @@ def finish_build(version_pk, build_pk, hostname=None, html=False,
         version.built = True
         version.save()
 
+    if not pdf:
```

Comment:
Seems like we should likely be clearing artifacts for all versions, when the use says to disable them. Though this might have some weird interactions if people accidently disable them, and then we blow away all their old tagged verisons, so I'm not sure the best course of action here.